Source/Core/Core/DSP/Jit/x64/DSPJitRegCache.cpp

```diff
@@ -194,7 +194,7 @@ DSPJitRegCache::DSPJitRegCache(DSPEmitter& emitter)
 
 DSPJitRegCache::DSPJitRegCache(const DSPJitRegCache& cache)
     : m_regs(cache.m_regs), m_xregs(cache.m_xregs), m_emitter(cache.m_emitter),
-      m_is_temporary(true), m_is_merged(false)
+      m_is_temporary(true), m_is_merged(false), m_use_ctr(0)
```

Comment:
Thoughts on moving everything that uses a constant value into the header (with direct initialization)?